calculator.js

```diff
@@ -0,0 +1,46 @@
+var calculator = {
+  value: 0,
```

Comment:
We mentioned this in the review session, but this `value` property  is *not* the same as the `value` variable used in the methods below. No worries, we'll cover how to make that connection this afternoon!